src/Mailgun/Mailgun.php

```diff
@@ -67,8 +67,10 @@ class Mailgun
      * @param string              $apiEndpoint
      * @param Hydrator|null       $hydrator
      * @param RequestBuilder|null $requestBuilder
+     *
+     * @internal Use Mailgun::configure or Mailgun::create instead.
      */
-    public function __construct(
+    public /* private */ function __construct(
```

Comment:
Please, do not comment code. If you think it is no longer useful, just drop it. If we need to restore the existing code then we can check the file history.